modules/orchestrator/src/main/java/org/eclipse/xpanse/modules/orchestrator/OrchestratorPlugin.java

```diff
@@ -54,4 +56,6 @@ public interface OrchestratorPlugin extends ServiceResourceHandler, Authenticati
      * @return true if all regions are valid.
      */
     boolean validateRegionsOfService(Ocl ocl);
+
+    Map<String, String> getResourcesInServiceDeployment(File scriptFile);
```

Comment:
since we are returning only the compute resources, it is better we rename this to `getComputeResourcesInServiceDeployment`